modules/update-conditional-web-console.adoc

```diff
@@ -22,7 +22,7 @@ You can view and assess the risks associated with particular updates with condit
 .Procedure
 
 . From the web console, click *Administration* -> *Cluster settings* page and review the contents of the *Details* tab.
-
+// update this to say "Include versions with known issues" in 4.17 once the UI catches up to the CLI
```

Comment:
This [already happened for 4.16][1], didn't it?

[1]: https://github.com/openshift/console/pull/13622#issue-2147083926

Reply:
@PratikMahajan mentioned that the UI work is occurring in 4.16. Can you confirm, @PratikMahajan ?

Reply:
My bad, it has already happened in 4.16

Reply:
Looks like it was covered now - will update!